frontend/lib/generate-image-markup.js

```diff
@@ -33,13 +34,11 @@ export function reStructuredText(badgeUrl, link, title) {
 }
 
 function quoteAsciiDocAttribute(attr) {
-  if (typeof attr === 'string') {
```

Comment:
Was this an unnecessary check before, or are now confident that only strings will be passed to this function?

Reply:
Yea, as far as I can tell, the only things passed in here will be strings (either empty or nonempty) that come from `this.state` in the badge customizer.